rspirv/README.md

```diff
@@ -34,7 +34,7 @@ First add to your `Cargo.toml`:
 
 ```toml
 [dependencies]
-rspirv = "0.11.0+sdk-1.2.198"
+rspirv = "0.11.0"
```

Comment:
This should also be gone automatically on the next `cargo-release` now that https://github.com/crate-ci/cargo-release/issues/308#issuecomment-905489443 is in and `{{version}}` excludes metadata, which is the substitute for https://github.com/crate-ci/cargo-release/pull/312 / #213.